src/exceptions.md

```diff
@@ -130,8 +131,8 @@ PROVIDE(PendSV = DefaultExceptionHandler);
 PROVIDE(SysTick = DefaultExceptionHandler);
 ```
 
-`PROVIDE` only takes effect when symbol on the RHS is still undefined after inspecting all the input
-object files. This is the scenario where the user didn't assign a handler to the exception.
+`PROVIDE` only takes effect when a symbol on the RHS is still undefined after inspecting all the input
```

Comment:
What is RHS here? Should it be LHS aka left-hand-side?

Reply:
Yes, I meant LHS. It may be better to be explicit and say "the symbol to the left of the equal sign".